### ISSUE TITLE
Support logFormat and logLevel for argocd-applicationset chart

### ISSUE THREAD

**Issue by anderius** [argoproj/argo-helm#790]
It would be nice if the argocd-applicationset could log json as argo-cd can.

**Comment by agaudreault-jive** [comment]
You can use that configuration in the value file. Requires v0.3.0
```yaml
  # -- Extra arguments to provide to the controller
  extraArgs:
    - '--logformat=json'
```

**Comment by jdomag** [comment]
@agaudreault-jive 
I set extraArgs which has been rendered properly (see below), but application set doesn't honor it. Still displaying info logs. I'm using v0.4.1 

![image](https://user-images.githubusercontent.com/28705876/158595376-d320f829-5553-4359-bea7-bb8a5944c39a.png)

**Comment by agaudreault-jive** [comment]
@jdomag I would say to open another bug for this. I think this is caused by the `debug` parameter being defined by default in the chart. Doc says it takes precedence, but it will always be defined. I think it should be removed from the chart by default, but others might have different opinions.

https://github.com/argoproj/applicationset/blob/22ddafc483de714a84b70e4dfae7f15ac1336689/main.go#L94

**Comment by rgarrigue** [comment]
@nhuray @agaudreault-jive is there a follow up bug here ? Using helm chart v4.3.0 with loglevel=warn & logformat=json we get this mixed output, json (as we want it) with text, still having info instead of the warn & above only

![image](https://user-images.githubusercontent.com/1872031/168761748-253fbf3f-359e-47d2-95fb-471b1921f09f.png)

And I don't see what I could tune to get this in line

**Comment by srosell-uz** [comment]
I think this has been fixed in the ArgoCD v2.5.0 release: https://github.com/argoproj/argo-cd/pull/10513

**Comment by rjchicago** [comment]
> I think this has been fixed in the ArgoCD v2.5.0 release: [argoproj/argo-cd#10513](https://github.com/argoproj/argo-cd/pull/10513)

Confirmed on `v2.6.6` & `v2.7.6` 👍 

I set the following in my `values.yaml`, having to include the settings as well in `applicationSet.extraArgs`:

``` yaml
argo-cd:
  global:
    logging:
      format: json
      level: error
  applicationSet:
    extraArgs:
    - --loglevel=error
    - --logformat=json
```
